rfc/async-first-team.md

```diff
@@ -0,0 +1,92 @@
+
+We formalize in this RFC the process for a thriving collaboration on the w3s team.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Out of scope
+
+* How to organize work
+* What templates to use for async updates and meetings
+
+## Team Background
+
+The w3s team comprises individuals who have played integral roles in the development of web3.storage and Filecoin Saturn, both initiatives deeply rooted within Protocol Labs. As an organization, Protocol Labs has embraced asynchronous communication since its inception, with half of our team boasting extensive tenures exceeding 5 years.
+
+With 12 members, our team predominantly consists of seasoned professionals with extensive expertise. Our team culture prioritizes shipping fast and iteratively without compromising quality, driven by a foundation of well-formulated proposals and a commitment to iterative feedback, discussions, and detailed specifications throughout the implementation process.
+
+Operating numerous services, our team is dispersed across multiple countries, leading to potential timezone variances of up to 10 hours. This global presence fosters around-the-clock productivity and a diverse perspective within our collaborative endeavors."
+
+## Async vs sync teams
+
+We recognize that each methodology offers its own set of advantages and drawbacks. Various teams may excel under different methodologies, influenced by factors such as team seniority, physical proximity, time zone disparities, and individual preferences.
+
+### Advantages of Async Team Communication
+
+1. Flexibility: Async communication allows team members to engage with work-related discussions and tasks at their convenience, accommodating diverse schedules and time zones.
+2. Reduced Interruptions: By minimizing synchronous meetings and real-time communication channels, async methodologies help mitigate interruptions, allowing individuals to focus on deep work and productivity.
+3. Increased Thoughtfulness: With async communication, individuals have more time to reflect on their responses and contributions, leading to more considered and well-articulated ideas. Moreover, by allowing participants to engage at their own pace and reducing the pressure of immediate responses, async methodologies create a more supportive environment where individuals with challenges such as social anxiety can contribute comfortably and constructively to team discussions.
+4. Documentation: Asynchronous communication encourages the documentation of discussions and decisions, fostering transparency and ensuring that information is accessible to all team members, as well as a ramp up for new members joining the team.
```

Comment:
I'd argue that async communication _requires_ good documentation to be effective, but does not necessarily _create_ it.  E.g. it is important to set best practices, resources and norms around what good documentation is (both content and format), in order to make async practices work better.